plugins/woocommerce-blocks/assets/js/blocks/collection-filters/edit.tsx

```diff
@@ -1,35 +1,62 @@
 /**
  * External dependencies
  */
-import { useBlockProps, useInnerBlocksProps } from '@wordpress/block-editor';
+import { InnerBlocks } from '@wordpress/block-editor';
 import { getSetting } from '@woocommerce/settings';
 import type { AttributeSetting } from '@woocommerce/types';
+import { Template } from '@wordpress/blocks';
+
+/**
+ * Internal dependencies
+ */
+import { EditProps, FilterType } from './types';
+import { getAllowedBlocks } from './utils';
+
+const DISALLOWED_BLOCKS = [
+	'woocommerce/filter-wrapper',
+	'woocommerce/collection-filters',
+];
 
 const ATTRIBUTES = getSetting< AttributeSetting[] >( 'attributes', [] );
 
 const firstAttribute = ATTRIBUTES.find( Boolean );
 
-const template = [
-	[ 'woocommerce/collection-active-filters', {} ],
-	[ 'woocommerce/collection-price-filter', {} ],
-	[ 'woocommerce/collection-stock-filter', {} ],
-	[ 'woocommerce/collection-rating-filter', {} ],
-];
+const templates: Partial< Record< FilterType, Template[] > > = {
+	'active-filters': [ [ 'woocommerce/collection-active-filters', {} ] ],
+	'price-filter': [ [ 'woocommerce/collection-price-filter', {} ] ],
+	'stock-filter': [ [ 'woocommerce/collection-stock-filter', {} ] ],
+	'rating-filter': [ [ 'woocommerce/collection-rating-filter', {} ] ],
+	'collection-filters': [
+		[ 'woocommerce/collection-active-filters', {} ],
+		[ 'woocommerce/collection-price-filter', {} ],
+		[ 'woocommerce/collection-stock-filter', {} ],
+		[ 'woocommerce/collection-rating-filter', {} ],
+		[ 'woocommerce/collection-attribute-filter', {} ],
+	],
+};
 
 if ( firstAttribute ) {
-	template.push( [
-		'woocommerce/collection-attribute-filter',
-		{ attributeId: parseInt( firstAttribute?.attribute_id, 10 ) },
-	] );
+	templates[ 'attribute-filter' ] = [
+		[
+			'woocommerce/collection-attribute-filter',
+			{ attributeId: parseInt( firstAttribute?.attribute_id, 10 ) },
+		],
+	];
 }
 
-const Edit = () => {
-	const blockProps = useBlockProps();
-	const innerBlockProps = useInnerBlocksProps( blockProps, {
-		template,
-	} );
+const Edit = ( props: EditProps ) => {
+	const allowedBlocks = getAllowedBlocks( DISALLOWED_BLOCKS );
+
+	const template = templates[ props.attributes.filterType ];
 
-	return <nav { ...innerBlockProps } />;
+	return (
+		<nav>
```

Comment:
We should keep the wrapper block props here
```suggestion
	const blockProps = useBlockProps();

	return (
		<nav { ...blockProps }>
```

Reply:
Whoops yup, thanks I'll apply this.